### DISCUSSION TITLE
Can I change the video quality settings Smart Cut?

### DISCUSSION THREAD

**Discussion by GitUser770** [mifi/lossless-cut#1920] [Support]
Hi all,

From what I gather the Smart Cut feature encodes the video parts which do not meet a keyframe exactly.
Is it possible to change the quality settings for the encoded parts?
I'm trying to cut with exact points which are ideally losslessly or as close to it as possible in terms of quality.
How can i achieve this?

Thank you.

**Comment by mifi** [comment]
hi. that is not currently possible. the bitrate is chosen automatically based on the bitrate of the input video. do you have a particular video that results in a bad quality encoded part?

related: #126
